codec/api/wels/codec_app_def.h

```diff
@@ -651,6 +651,9 @@ typedef struct {
   EVideoFrameType eFrameType;
   int iFrameSizeInBytes;
   long long uiTimeStamp;
+  unsigned char*  pReconData[3];   ///< plane data of reconstructed frame
```

Comment:
I think we should either reuse `SSourcePicture` or define a new structure that can provide the correct information about colorspace, size, planes and strides for the recon frame.